### ISSUE TITLE
glyma.Zh35_IGA1004.gnm1.ann1.RGN6.gene_models_main.gff3 lacks ID attribute

### ISSUE THREAD

**Issue by sammyjava** [legumeinfo/datastore-issues#83]
```
Retrieving Zh35_IGA1004.gnm1.ann1.RGN6.gene_models_main in a tgt items database
[convertFile] Processing README.Zh35_IGA1004.gnm1.ann1.RGN6.yml
[convertFile] Processing glyma.Zh35_IGA1004.gnm1.ann1.RGN6.gene_models_main.gff3

> Task :dbmodel:retrieveSingleSource FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':dbmodel:retrieveSingleSource'.
> java.lang.RuntimeException: GFF line does not include ID: glyma.Zh35_IGA1004.gnm1.chr01       exonerate       CDS     -3331   -2671   0.0     0       Parent=glyma.Zh35_IGA1004.gnm1.ann1.SoyZH35_01R003120.1
```

**Comment by sammyjava** [comment]
@adf-ncgr could you hit this soon, as I've got the rest of Zh35_IGA1004 loaded and it'll soon be the only thing left to do before post-processing. Thanks.

**Comment by adf-ncgr** [comment]
looks like I had fixed this in #27 then it got clobbered when functional descriptors were added. re-fixed using same protocol as before, try it now.

**Comment by sammyjava** [comment]
Loaded, thanks!
